### ISSUE TITLE
Releasing a version of ember-cli-mirage

### ISSUE THREAD

**Issue by cah-briangantzler** [miragejs/ember-cli-mirage#2055]
This issue covers questions on how a release of ember-cli-mirage is performed.

I watched the video at https://www.youtube.com/watch?v=IeKNE_ERadY.  Very Helpful. 

I looked over the travis config, ember-cli-deploy config and I think I pieced together all the things that are happening (wanted to know more behind the scenes to apply these things to my own repos)

When I go to https://www.ember-cli-mirage.com/versions/master/ I get the below error in the console and the page does not work for the master branch only. All other versions work. Do you see this error? does the page work for you? Dont want to release a broken docs page 

![image](https://user-images.githubusercontent.com/1529286/94696233-d1d3c000-0304-11eb-8f36-9372bc98a973.png)

**Comment by samselikoff** [comment]
Hm good catch! Merging some of the Dependabot PRs, lets see if any of the dependency updates fixes the problem. If not I'm guessing it's related to ember data / fastboot weirdness.

**Comment by cah-briangantzler** [comment]
K, Ill check the docs in a few hours and see if the same issue is happening

**Comment by cah-briangantzler** [comment]
unfortunately still getting the above error on the master branch

**Comment by samselikoff** [comment]
will take another look today, next release is going to be major so we can bump all these deps. Only breaking change is node 8.

**Comment by samselikoff** [comment]
Ok updates are all passing on travis but master is still broken, I think the next step would be to upgrade addon docs.

Here's the PR, it's failing right now: https://github.com/miragejs/ember-cli-mirage/pull/2023

So that would be the next thing to investigate

Here's AD changelog: https://github.com/ember-learn/ember-cli-addon-docs/blob/master/CHANGELOG.md

Maybe we need to bump Ember Source but don't know without looking into it more.

**Comment by cah-briangantzler** [comment]
Weird that its only failing on 3.18.  I will take a look and see if I can find the problem.

**Comment by cah-briangantzler** [comment]
Any thoughts on removing the acceptance test since its not testing the app but instead testing that the docs show? This addon supports 2.18, but addon docs no longer does.

**Comment by samselikoff** [comment]
Yeah, or is there a way to get it to only run unversioned? Only on the latest ember (the one from package.json)? That test has definitely helped a few times

**Comment by cah-briangantzler** [comment]
I am unaware of a way to conditionally run a test like that

**Comment by cah-briangantzler** [comment]
Even with the addon docs updates now, master is still a broken docs. I dont see what the problem is. Should I go ahead with attempting the release and just hope its a master branch only thing?

**Comment by samselikoff** [comment]
Hm I would say it'd be best to get it working, otherwise I think the site will be broken as soon as we publish.

Is it a lockfile thing? Maybe you just need to re-generate the lockfile. Or pin ember to a certain version in package.json.

**Comment by cah-briangantzler** [comment]
Not sure how to test and find the problem. The master branch docs work fine locally, its only on the website that it doesnt work. To test that I have to have PR merged in order to see the results. Any idea how else to test it?

**Comment by samselikoff** [comment]
Oh - but didn't we merge the pr? So isn't master on prod the same as master in the repo?

The build that is being served on [https://www.ember-cli-mirage.com/versions/master/](https://www.ember-cli-mirage.com/versions/master/) is [here](https://github.com/miragejs/ember-cli-mirage/tree/gh-pages/versions/master), so you could check out the `gh-pages` branch, `cd` into that directory and run `python -m SimpleHTTPServer` to see if the problem also happens on your comp. If it does you could try `ember build` then same thing to go from dev to prod to see if you can reproduce.

**Comment by cah-briangantzler** [comment]
K, will try that.

**Comment by cah-briangantzler** [comment]
TLDR  Upgrade ember-cli-addon-docs to latest release.  https://github.com/miragejs/ember-cli-mirage/pull/2081


So the gh-pages fails when selecting master, no surprise there, I am not real familiar with publishing ember-cli-addon-docs.

so from the master branch I did an `ember build` then I think you wanted me to do an `ember deploy production`. That didnt work, I tried several things, they all resulted in the same error.   Reading the ember-cli-docs, the master branch isnt tagged, so I create a branch off master called `docs-test` and the addon docs said it would create a version off the name of my branch, still same error. It errors really quickly with the follow:

`
Deploying [🚀 -------] 13% [plugin: addon-docs -> setup]Error: Command failed with exit code 1: git fetch --all
Error: Command failed with exit code 1: git fetch --all
    at makeError (/Users/brian.gantzler/source-personal/ember-cli-mirage/node_modules/ember-cli-addon-docs/node_modules/execa/lib/error.js:56:11)
    at handlePromise (/Users/brian.gantzler/source-personal/ember-cli-mirage/node_modules/ember-cli-addon-docs/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)Pipeline aborted
`

Since im unfamiliar with the process of creating the addon docs site, not sure where to go from here. 


I then went back and checked out the commit on the gh-pages branch that was the publish of 1.1.8 and the master version of the gh-pages worked, I then went through every commit to see where it broke and it broke on the very first commit of gh-pages after 1.1.8 which would be this commit https://github.com/miragejs/ember-cli-mirage/commit/4240f6f0394c35e70b2f0e2a67bd03134a1f3040 which really has nothing to do with it, but selecting the master version of the addon docs has been broken since then.  

The one that worked was on 8/6 the one that failed was on 8/10 which leads me to this issue I opened on 8/7 https://github.com/ember-cli/ember-fetch/issues/547 

We had a dependabot update for addon docs to 0.8.0 and hoped that would fix it, but that was released 8/1 and 0.9.0 was released 8/10 that fixed this issue. https://github.com/ember-learn/ember-cli-addon-docs/blob/master/CHANGELOG.md#v090-2020-08-10 Dependabot has not submitted a request to update addon docs since then, not sure why

**Comment by Turbo87** [comment]
v2.0.0 has been released now 🎉

**Comment by samselikoff** [comment]
Thanks again!
